### ISSUE TITLE
Establish preprocess pipeline

### ISSUE THREAD

**Issue by timothyas** [NOAA-PSL/graph-ufs#45]
Create prototypes/p1/preprocess.py that does the following:

1. Pull in remote training data:
    * Store as stacked inputs.zarr, targets.zarr
2. Pull in normalization statistics:
    * Store as stacked statistics.zarr 
3. Compute or manage the variable and level weighting for loss function
    * TBD if this needs to be stored.

... anything else?
